test/per_file_coverage.sh

```diff
@@ -9,18 +9,17 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/common:96.3"
 "source/common/common/posix:94.1"
 "source/common/crypto:0.0"
-"source/common/event:93.4" # Emulated edge events guards don't report LCOV
+"source/common/event:94.2" # Emulated edge events guards don't report LCOV
 "source/common/filesystem/posix:96.2"
-"source/common/http/http3:93.9"
 "source/common/json:90.9"
 "source/common/network:95.0"  # Flaky, `activateFileEvents`, `startSecureTransport` and `ioctl` do not always report LCOV
 "source/common/protobuf:94.8"
 "source/common/signal:84.5" # Death tests don't report LCOV
 "source/common/singleton:95.1"
 "source/common/thread:0.0" # Death tests don't report LCOV
 "source/common/matcher:93.3"
-"source/common/quic:87.4"
-"source/common/tracing:94.9"
+"source/common/quic:88.4"
+"source/common/tracing:95.7"
```

Comment:
Turns out 88.4 was too high... see line 17696... https://dev.azure.com/cncf/4684fb3d-0389-4e0b-8251-221942316e06/_apis/build/builds/76091/logs/157

Reply:
It wasn't at the time, it was a merge conflict with https://github.com/envoyproxy/envoy/pull/16534 which disabled tests for the coverage build.
Have PRs out to fix though!